codice/ddf
# DDF-04485 Migrate Visualization Selector to React

#### What does this PR do?
Migrates visualization-selector.view.js to react
#### Who is reviewing it? 
@willwill96 
@mojogitoverhere 
@rymach 

#### Select relevant component teams: 
@codice/ui 
#### Ask 2 committers to review/merge the PR and tag them here.

@andrewkfiedler 
@djblue
#### How should this be tested?
full build of ui directory
test the visualization-selector (bottom right of the map) 
select all the different visualizations doing a combo of drag and drop or just clicking
ensure no regressions

#### Any background context you want to provide?
#### What are the relevant tickets?
For GH Issues:
Fixes: #4485 

#### Screenshots

#### Checklist:

#### Notes on Review Process
Please see [Notes on Review Process](https://codice.atlassian.net/wiki/spaces/DDF/pages/71946981/Pull+Request+Guidelines) for further guidance on requirements for merging and abbreviated reviews. 

#### Review Comment Legend:
- ✏️ (Pencil) This comment is a nitpick or style suggestion, no action required for approval. This comment should provide a suggestion either as an in line code snippet or a gist. 
- ❓ (Question Mark) This comment is to gain a clearer understanding of design or code choices, clarification is required but action may not be necessary for approval.
- ❗ (Exclamation Mark) This comment is critical and requires clarification or action before approval.
